### DISCUSSION TITLE
Import stl or any other mesh format

### DISCUSSION THREAD

**Discussion by cemag1** [JWock82/Pynite#165] [Ideas]
Was wondering if there is scope (or already implemented) to import any mesh format and conduct some FEA on it. Nevertheless excellent work, thanks for making open-source!

**Comment by JWock82** [comment]
I believe the STL format uses triangular elements in the mesh. `Pynite` does not have triangular elements. I've thought about adding them to allow for more arbitrary mesh shapes.

Is there any format for quad meshes you'd find useful to read in?

**Reply by cemag1** [reply]
Thanks for the reply. Supporting triangular elements would be so convenient!! Are you aware of any quad mesh formats that I could check out?
